modules/perception/camera/app/lane_camera_perception.cc

```diff
@@ -69,12 +69,13 @@ void LaneCameraPerception::InitLane(
     const std::string &work_root, base::BaseCameraModelPtr &model,
     const app::PerceptionParam &perception_param) {
   // Init lane
-  CHECK(perception_param.has_lane_param()) << "Failed to include lane_param";
-  {
+  CHECK(perception_param_.lane_param_size() > 0)
+      << "Failed to include lane_param";
+  for (int i = 0; i < perception_param_.lane_param_size(); ++i) {
     // Initialize lane detector
-    auto lane_param = perception_param.lane_param();
+    auto lane_param = perception_param.lane_param(i);
```

Comment:
1. Use reference if possible, to avoid making a copy.
2. Use const if possible.

Reply:
Agreed. 
Changed "auto" to "auto const &" in both lane_camera_perception.cc and obstacle_camera_perception.cc